src/selectors/clusters.tsx

```diff
@@ -55,6 +55,9 @@ const clusterToClusterTableRow = (cluster: Cluster): IRow => {
   };
 };
 
+export const getClusterTableStatusCell = (rowData: IRowData) =>
+  rowData?.cells?.[3] as HumanizedSortable;
```

Comment:
We should start to be careful when referencing cells. The table structure seems to be not stable as expected to be at the beginning.